### PR TITLE
Button

### PR DESCRIPTION
**Pull request by NagaSatyaGeeth**
add button component
https://4vkgr.csb.app/
button

### CODE REVIEW

File: button/button.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                <li class="side-bar-item"><a href="#">Image</a></li>
+                <li class="side-bar-item"><a href="#">Input</a></li>
+                <li class="side-bar-item"><a href="#">Text Utilities</a></li>
+                <li class="side-bar-item"><a href="#">Lists</a></li>
+                <li class="side-bar-item"><a href="#">Navigation</a></li>
+                <li class="side-bar-item"><a href="#">Modal</a></li>
+                <li class="side-bar-item"><a href="#">Rating</a></li>
+            </ul>
+          </aside>
+    </div>
```
Comment (shaik-saad) [inline button/button.html]: You can remove this parent div to aside, as aside is semantic.

**Review by shaik-saad — APPROVED** [review]
You can use proper indentation as it increases readability, other than that you're good to merge.

**Comment by shaik-saad** [comment]
BTW, I liked the brand name and your brand logo.

**Review by Sudhanshusoni45 — APPROVED** [review]
Good to go!

Just a suggestion, you can reduce box-shadow of buttons they are seeming a bit large.

**Review by SubhamChoudhury — APPROVED** [review]

**Comment by NagaSatyaGeeth** [comment]
> Good to go!
> 
> Just a suggestion, you can reduce box-shadow of buttons they are seeming a bit large.

fixed this

**Pull request merged**